# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../luajit/install-symlink.patch
# Copyright (C) 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- LuaJIT-2.1.0-beta3/Makefile.vanilla	2020-09-21 10:50:31.746699094 +0000
+++ LuaJIT-2.1.0-beta3/Makefile	2020-09-21 10:50:40.614699098 +0000
@@ -135,7 +135,7 @@
 	@echo "Note: the development releases deliberately do NOT install a symlink for luajit"
 	@echo "You can do this now by running this command (with sudo):"
 	@echo ""
-	@echo "  $(SYMLINK) $(INSTALL_TNAME) $(INSTALL_TSYM)"
+	$(SYMLINK) $(INSTALL_TNAME) $(INSTALL_TSYM)
 	@echo ""
 
 
